frontend/packages/integration-tests-cypress/tests/app/demo-dynamic-plugin.spec.ts

```diff
@@ -0,0 +1,198 @@
+import { safeLoadAll } from 'js-yaml';
+import { checkErrors } from '../../support';
+import { isLocalDevEnvironment } from '../../views/common';
+import { detailsPage } from '../../views/details-page';
+import { infoMessage, hintMessage, warningMessage, warningToast } from '../../views/form';
+import { listPage } from '../../views/list-page';
+import { modal } from '../../views/modal';
+import { nav } from '../../views/nav';
+
+const PLUGIN_NAME = 'console-demo-plugin';
+const PLUGIN_PATH = '../../../dynamic-demo-plugin';
+const CHECK_UPDATE_WAIT = 30000;
+const PLUGIN_PULL_SPEC = Cypress.env('PLUGIN_PULL_SPEC');
+
+/*
+  These tests are meant to:
+    1. show how to test a dynamic plugin using demo as the plugin instance
+    2. run locally:
+      2a. build the plugin locally, and run the server
+      2b. using bridge running with the plugin argument that points to the local dynanmic plugin server
+      2c. will intercept the check update call to mock a toast notification that a plugin has been enabled
+      2d. will not use all workload definitions defined in the yaml (not using the env variable for pull spec)
+    3. run on ci:
+      3a. ci will build the dynamic plugin and provide the pullspec in the env var: CYPRESS_PLUGIN_PULL_SPEC
+      3b. that pull spec will be used to create the deployment on the cluster
+      3c. intercepting the check-updates call will not occur as the toast will appear when the plugin is enabled
+    4. the scaffolding should remain the same except modifying the constants above
+ */
+
+const enableDemoPlugin = (enable: boolean) => {
```

Comment:
Can we enable plugin via CLI?  that would greatly simplify the process and code..

Reply:
@yapei we are creating our workloads via the oc create cli commands. We want to test the gui and the ability to set up these plugins via the gui and not the cli. I think here we want to make sure we are testing the gui.